Policies/Kubernetes/container-no-privilege-escalation/template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazurecontainernoprivilegeescalation
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureContainerNoPrivilegeEscalation
```

Comment:
Don't we need to specify listKind, singular, plural here? Please check with Sertac or Rita.

Reply:
That requirement was removed a while ago. Gatekeeper now infers those values

Reply:
@sozercan Can you double confirm this?

Reply:
that's correct, those were removed from gk library also